designs/2024-plugins.eslint.org-website/README.md

```diff
@@ -151,6 +158,8 @@ Not relevant for this proposal.
 
 Any help in defining the details of this change or with the subsequent implementation will be greatly appreciated.
 
+To find out which topics will need help exactly, we need answer the open questions in this RFC first.
```

Comment:
My expectation when you agreed to put together the RFC is that you'd come up with an initial design based on your understanding of the problem and we could iterate on it together. It looks like this is still just a list of questions. What do you think about closing this PR until you've had time to come up with a concrete proposal, rather than leaving this open?

I'd much rather have you attempt to answer the open questions and come up with an initial proposal than require all of us to keep going around on open questions to make any progress.

Reply:
Thanks for the feedback. I was trying to collect input based on the experience of others who already worked with the eslint.org website and the tools we use there, but I can see that this is just a bit too abstract to engage users in a discussion. I agree it makes sense to close this RFC at this point, and I will try to come up with a design and likely a prototype of how the plugins website will work. Then I will open a new RFC to discussion the suggested changes.